webapp/tests/Unit/Service/ImportExportServiceTest.php

```diff
@@ -43,63 +52,72 @@ public function testImportContestDataErrors(mixed $data, string $expectedMessage
     public function provideImportContestDataErrors(): Generator
     {
         yield [[], 'Error parsing YAML file.'];
-        yield [['name' => 'Some name'], 'Missing fields: one of (start_time, start-time), one of (id, short_name, short-name), duration'];
-        yield [['short-name' => 'somename', 'start-time' => '2020-01-01 12:34:56'], 'Missing fields: one of (name, formal_name), duration'];
+        yield [
+            ['name' => 'Some name'],
+            'Missing fields: one of (start_time, start-time), one of (id, short_name, short-name), duration',
+        ];
+        yield [
+            ['short-name' => 'somename', 'start-time' => '2020-01-01 12:34:56'],
+            'Missing fields: one of (name, formal_name), duration',
+        ];
         yield [
             [
-                'name'       => 'Test contest',
+                'name' => 'Test contest',
                 'short-name' => 'test',
-                'duration'   => '5:00:00',
+                'duration' => '5:00:00',
                 'start-time' => 'Invalid start time here',
             ],
-            'Can not parse start-time'
+            'Can not parse start-time',
         ];
         yield [
             [
-                'name'       => 'Test contest',
-                'id'         => 'test',
-                'duration'   => '5:00:00',
+                'name' => 'Test contest',
+                'id' => 'test',
+                'duration' => '5:00:00',
                 'start_time' => 'Invalid start time here',
             ],
-            'Can not parse start_time'
+            'Can not parse start_time',
         ];
         yield [
             [
-                'name'                     => 'Test contest',
-                'short-name'               => 'test',
-                'duration'                 => '5:00:00',
-                'start-time'               => '2020-01-01T12:34:56+02:00',
+                'name' => 'Test contest',
```

Comment:
Is there anything how we can disable those changes for your editor? In this case I find this harder to read and I think PSR has no opinion about this kind of indentation?

Reply:
This is me manually pressing a button to auto format a file before save, which is baked into my muscle memory, since I like it. I will undo this for this file.